WMF Framework/Did You Know/WMFFeedDidYouKnowFetcher.swift

```diff
@@ -0,0 +1,64 @@
+import WMFData
+
+public final class WMFFeedDidYouKnowFetcher: Fetcher {
+    private let userDefaultsStore = WMFDataEnvironment.current.userDefaultsStore
+
+    public func fetchDidYouKnow(withSiteURL siteURL: URL, completion: @escaping (Error?, [WMFFeedDidYouKnow]?) -> Void) {
+
+        let sharedCache = SharedContainerCache.init(fileName: self.cachedFileName(for: siteURL), subdirectoryPathComponent: SharedContainerCacheCommonNames.didYouKnowCache)
```

Comment:
I don't think this cache is working - I see the dyk endpoint attempting to fetch every time.